ksqldb-execution/src/main/java/io/confluent/ksql/execution/function/UdtfUtil.java

```diff
@@ -41,10 +42,12 @@ public static KsqlTableFunction resolveTableFunction(
 
     final List<Expression> functionArgs = functionCall.getArguments();
 
-    final List<SqlType> argTypes = functionArgs.isEmpty()
-        ? ImmutableList.of(FunctionRegistry.DEFAULT_FUNCTION_ARG_SCHEMA)
-        : functionArgs.stream().map(expressionTypeManager::getExpressionSqlType)
-            .collect(Collectors.toList());
+    final List<SqlArgument> argTypes = functionArgs.isEmpty()
+        ? ImmutableList.of(SqlArgument.of(FunctionRegistry.DEFAULT_FUNCTION_ARG_SCHEMA))
+        : new ArrayList<>();
```

Comment:
Same as above, for a minute I couldn't figure out how to get the additional `map(arg -> sqlArgument.of(arg.sqlType())` to work, but we should be able to add that to the original implementation and avoid the for loop